packages/babel-plugin-proposal-optional-chaining/test/fixtures/transparent-expr-wrappers/ts-parenthesized-expression-member-call/input.ts

```diff
@@ -0,0 +1 @@
+((o?.Foo.m) as ExampleType)()
```

Comment:
Nit: please add a trailing new line in these fails :pray:

Reply:
Of course, done!